### ISSUE TITLE
Add option to skip first call of a function

### ISSUE THREAD

**Issue by timabbott** [pyutils/line_profiler#20]
A common situation when profiling is that a piece of code has a high first-call cost (e.g. the first call will do some expensive initialization), but future calls to the piece of code will hopefully be fast because they can reuse that initialization. It'd be awesome if there were a way to pass an argument to LineProfiler to skip the first run of a given function for the purposes of the statistics. E.g. syntax like this:

```
import line_profiler                                                                                                                        
profile = line_profiler.LineProfiler()                                                                                                      
                                                                                                                                            
@profile(skip_first_call=True)
```

(I'm glad to see this project is maintained again!)

**Comment by Nodd** [comment]
Duplicate of #30 (which came later but has more comments)

**Comment by Erotemic** [comment]
Good point, closed in favor of #30
